ztp/gitops-subscriptions/argocd/WorkerNodeDeletion.md

```diff
@@ -0,0 +1,52 @@
+# Delete and re-provision a worker node using ZTP
+Starting from ACM 2.8, it supports GitOps workflow to cleanly delete a node from an existing cluster by deleting the BMH CR on the hub cluster that is annotated for cleanup. This document provides guidance on how to delete and re-provision a worker node using ZTP workflow.
+
+## Prerequisites
+1. A spoke cluster (ie. Standard, Compact+workers, sno+workers) installed and configured using the GitOps ZTP flow, as described in [GitOps ZTP flow](README.md)
+1. ACM 2.8+ with MultiClusterHub created and configured, running on OCP 4.13+ bare metal cluster
+
+## Delete a worker node from spoke cluster
+1. Annotate the BMH CR of the worker node with the "bmac.agent-install.openshift.io/remove-agent-and-node-on-delete=true"    annotation. Add the annotation via SiteConfig as the following, then push the changes to git repo and wait for the BMH CR on the hub cluster has the annotation applied.
+    ```yaml
+    nodes:
+    - hostname: node6
+      role: "worker"
+      crAnnotations:
+        add:
```

Comment:
Why do we need to mentioned "add" ? can we remove "add" and become like 
` BareMetalHost:`
`    bmac.agent-install.openshift.io/remove-agent-and-node-on-delete: true`